front/lib/api/assistant/configuration.ts

```diff
@@ -118,6 +118,7 @@ function determineGlobalAgentIdsToFetch(
     case "list":
     case "all":
     case "admin_internal":
+    case "manage-assistants-search":
```

Comment:
Isn't this the same behaviour as "list" then ? why do we need another view ?

Reply:
no, we need 1. all global agents even those inactive, 2. all published agents even those not in user's list